website/guides/border.md

```diff
@@ -0,0 +1,164 @@
+# adding an sgb border to your non-sgb game
```

Comment:
```suggestion
# Adding a SGB border to your non-SGB game
```

Reply:
"an ess-jee-bee" or "a Super Game Boy"?

Reply:
i intentionally used an initialism there - it should be "an SGB".